pyproject.toml

```diff
@@ -43,4 +43,7 @@ lint.ignore = [
   "PD011", # pandas-use-of-dot-values
   "PLR0913", # too-many-arguments in function
   "S101", # Using assert
+  "PLR2004", # magic-value-comparison
+  "N802", # snake_case function naming convention
```

Comment:
I checked and it seems not possible to define different ignores for different folders when using ruff. Therefore, I think N802 should be added on a local basis, i.e., on the line where it is needed. Otherwise, we do not enforce the naming convention for the non-test folders.